app/src/main/java/de/danoeh/antennapod/fragment/QueueFragment.java

```diff
@@ -496,6 +497,9 @@ private void reallyMoved(int from, int to) {
 
         txtvEmpty = root.findViewById(android.R.id.empty);
         txtvEmpty.setVisibility(View.GONE);
+        txtvHeadEmpty = root.findViewById(R.id.emptyQueueHeader);
+        txtvHeadEmpty.setVisibility(View.GONE);
```

Comment:
I think it would be better to wrap the empty textViews in a LinearLayout. That way, you only have to hide one single element. Additionally, it allows to add an icon later without modifying the code here again.